src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -110,14 +112,36 @@ const uint8_t MPU9250::_checked_registers[MPU9250_NUM_CHECKED_REGISTERS] = { MPU
 									   };
 
 
+const uint16_t MPU9250::_icm20948_checked_registers[ICM20948_NUM_CHECKED_REGISTERS] = {
```

Comment:
Why was this added?

Reply:
The ICM20948 uses different register addresses for the same/similar functions as the MPU9250, so another set of checked registers is needed. Also the ICM distributes its registers accross 4 register banks, so every register is 1 byte address + 2 bit information for its bank. I put both into the now 16bit address value as the write_reg/write_checked_reg functions use 16 bit integer addresses already to encode the bus speed, and the register bank value now goes alongside.